<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QWidget" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SaleInvoice</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>730</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>15</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Ready To Print ...</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Y_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>180</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Images/yash_logo.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>95</y>
     <width>1000</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Shop Name :</string>
   </property>
  </widget>
  <widget class="QFrame" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>960</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QFrame" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>312</y>
     <width>960</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QFrame" name="line_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>3</width>
     <height>220</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::VLine</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QFrame" name="line_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>960</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QFrame" name="line_7">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>120</y>
     <width>3</width>
     <height>220</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::VLine</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QFrame" name="line_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>3</width>
     <height>460</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::VLine</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QFrame" name="line_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>760</y>
     <width>960</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QFrame" name="line_11">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>340</y>
     <width>3</width>
     <height>460</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::VLine</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Shop Address: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="printButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>810</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PRINT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveWithoutPrintingButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>810</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SELL WITHOUT PRINTING</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>180</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Date: </string>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>180</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="shopNameLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>251</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Yash Agency &amp; Chaitanya Enterprise</string>
   </property>
  </widget>
  <widget class="QLabel" name="shopAddressLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>251</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>123, Phaltan, Maharashtra</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="invoiceTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>339</y>
     <width>940</width>
     <height>301</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>#</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Item</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Unit Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>831</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Grand Total: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>690</y>
     <width>831</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Payment: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>730</y>
     <width>831</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Amount Due: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="grandTotalLabel">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>650</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="paymentLabel">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>690</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="amountDueLabel">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>730</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="amountDueLabel_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>770</y>
     <width>351</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Thank you for business</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="invoiceNoLabel">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>130</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>130</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Invoice No. :</string>
   </property>
  </widget>
  <widget class="QLabel" name="customerNameLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Customer Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="phoneNoLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Phone No. :</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>800</y>
     <width>960</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
